appcues/src/main/java/com/appcues/trait/appcues/TooltipsPaths.kt

```diff
@@ -163,89 +198,166 @@ private fun getTooltipPointerPath(
     animationSpec: AnimationSpec<Float>,
 ): Path {
     val density = LocalDensity.current
+    val cornerRadiusPx = with(density) { containerDimens.cornerRadius.toPx() }
 
-    val pointerOffsetX = calculatePointerXOffset(
+    val (pointerOffsetX, verticalTipOffset) = calculatePointerXOffset(
         containerDimens = containerDimens,
         tooltipSettings = tooltipSettings,
         pointerOffsetX = with(density) { tooltipSettings.pointerOffsetX.value.toPx() },
-        cornerRadius = with(density) { containerDimens.cornerRadius.toPx() },
+        cornerRadiusPx = cornerRadiusPx,
+    )
+
+    val (pointerOffsetY, horizontalTipOffset) = calculatePointerYOffset(
+        containerDimens = containerDimens,
+        tooltipSettings = tooltipSettings,
+        pointerOffsetY = with(density) { tooltipSettings.pointerOffsetY.value.toPx() },
+        cornerRadiusPx = cornerRadiusPx,
+    )
+
+    val tipLength = animateFloatAsState(
```

Comment:
moving the animated tip length here since has similar logic untied to being horizontal or vertical positioning